docs/docs/References/Plugin_Operations.md

```diff
@@ -1232,20 +1228,20 @@ A Virtual Source [Data Migration](Glossary.md#data-migration) transforms virtual
 
 Argument | Type | Description
 -------- | ---- | -----------
-migration_id | String | The ID of this migration. An ID is a string containing one or more positive integers separated by periods. Each ID must be unique.
+migration_id | String | The ID of this migration. An ID is a string containing one or more positive integers separated by periods. Each ID must be unique. More details [here](/Versioning_And_Upgrade/Upgrade.md#rules-for-data-migrations).
 
 ### Function Arguments
 Argument | Type | Description
 -------- | ---- | -----------
-old_virtual_source | Dictionary | The plugin-specific data associated with a virtual source, in an old format.
+old_virtual_source | Dictionary | The plugin-specific data associated with a virtual source, that conforms to the previous schema.
 
 !!! warning
-    The incoming data is a Python dictionary, where each property name appears exactly as described in the schema. This differs from non-upgrade-related operations, where the incoming data uses autogenerated classes.
+    The function argument `old_virtual_source ` is a Python dictionary, where each property name appears exactly as described in the previous virtual source schema. This differs from non-upgrade-related operations, where the function aguments are [autogenerated classes](Schemas_and_Autogenerated_Classes.md) based on the schema.
```

Comment:
nit: ditto - "aguments" typo and extra space after `old_virtual_source`

Reply:
fixed